tests/test_ocs_basic_install.py

```diff
@@ -181,3 +275,84 @@ def load_config_data(data_path):
     """
     with open(data_path, "r") as data_descriptor:
         return yaml.load(data_descriptor, Loader=yaml.FullLoader)
+
+
+def create_rook_resource(template_name, rook_data, cluster_path):
```

Comment:
I would rename this function to something like create_oc_resource as it's about communicate with OCS, and would make the template path parametrized.

Move it to some place to be able to be easily used by others.

Reply:
Fixed